roxie/ccd/ccdfile.cpp

```diff
@@ -389,7 +389,7 @@ class CRoxieLazyFileIO : implements ILazyFileIO, implements IDelayedFile, public
     virtual const char *queryFilename() { return logical->queryFilename(); }
     virtual bool isAlive() const { return CInterface::isAlive(); }
 
-    virtual IMemoryMappedFile *queryMappedFile()
+    virtual IMemoryMappedFile *getMappedFile() override
```

Comment:
style: If it is called getX() it should link the value it returns.  This is returning the pointer unlinked - assuming it should be linked.